src/osmet/mod.rs

```diff
@@ -74,8 +74,20 @@ struct Osmet {
     size: u64,
 }
 
+#[derive(Serialize)]
+struct FiemapOutput {
+    extents: Vec<Extent>,
+}
+
 pub fn osmet_fiemap(config: &OsmetFiemapConfig) -> Result<()> {
-    eprintln!("{:?}", fiemap_path(config.file.as_str().as_ref())?);
+    let output = FiemapOutput {
+        extents: fiemap_path(config.file.as_str().as_ref())?,
+    };
+    serde_json::to_writer_pretty(std::io::stdout(), &output)
+        .context("failed to serialize extents")?;
+    std::io::stdout()
+        .write_all(b"\n")
```

Comment:
Doesn't matter here, but I would use [the lock method](https://doc.rust-lang.org/std/io/struct.Stdout.html#method.lock).  It can start to matter when doing nontrivial amounts of I/O, and it's not a difficult pattern, so better to use it consistently.

Reply:
:+1: We don't generally do that now; will leave it for a followup.

Reply:
https://github.com/coreos/coreos-installer/pull/618